cli/QuickstepCli.cpp

```diff
@@ -430,6 +445,13 @@ int main(int argc, char* argv[]) {
         reset_parser = true;
         break;
       }
+#ifdef QUICKSTEP_ENABLE_GOOGLE_PROFILER
+      // Profile only if profile_file_name flag is set
+      if (!started_profiling && !quickstep::FLAGS_profile_file_name.empty()) {
```

Comment:
It’s at the top, below the definition of the flag. It’s more useful there, since it’s really a comment about how to use the flag. But it’s too long to be part of the flag description itself.

> On Jun 13, 2016, at 16:18, Zuyu ZHANG <notifications@github.com> wrote:
> 
> In cli/QuickstepCli.cpp <https://github.com/apache/incubator-quickstep/pull/28#discussion_r66870133>:
> 
> > @@ -430,6 +445,13 @@ int main(int argc, char* argv[]) {
> >          reset_parser = true;
> >          break;
> >        }
> > +#ifdef QUICKSTEP_ENABLE_GOOGLE_PROFILER
> > +      // Profile only if profile_file_name flag is set
> > +      if (!started_profiling && !quickstep::FLAGS_profile_file_name.empty()) {
> @navsan <https://github.com/navsan> I did not see the detailed explanation beyond Profile only if profile_file_name flag is set. Did I miss something?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/apache/incubator-quickstep/pull/28/files/00955f19652fe54468377e932365d77a13cc69c8#r66870133>, or mute the thread <https://github.com/notifications/unsubscribe/ACZB66ciwFxRTrf6q0ZQ4aw1JZqljhuQks5qLckqgaJpZM4Iz4mC>.
>